.github/workflows/post_to_mastodon.yml

```
@@ -0,0 +1,31 @@
+  post_to_mastodon:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    steps:
+      - name: Post to Mastodon
+      - uses: nick-fields/retry@v2
+        with:
+          timeout_minutes: 2
+          max_attempts: 3
+          command: |
+          curl -X POST -H "Authorization: Bearer ${{ secrets.MASTODON_ACCESS_TOKEN }}" \
+          -F "status=Pull request #${{ github.event.pull_request.number }} merged: ${{ github.event.pull_request.title }}" \
```

actionlint

could not parse as YAML: yaml: line 25: could not find expected ':'

(syntax-check)